app/src/main/java/lt/vilnius/tvarkau/fragments/BaseFragment.kt

```diff
@@ -53,7 +57,8 @@ abstract class BaseFragment : Fragment() {
 
     override fun onDestroyView() {
         super.onDestroyView()
-        refWatcher.watch(this)
+        //Something is wrong. Need to discuss this more...
+//        refWatcher.watch(this)
```

Comment:
I don't understand why `refWatcher.watch(this)` should be attached `onDestroyView()`.
That way it will always spew out reports, that fragment is leaking EventBus, because it's purpose is to keep fragment and receive events even when views are destroyed, but fragment is not.